### PR TITLE
Manager Camera now has Following!

### PR DESCRIPTION
**Pull request by InsightfulParasite**
## About The Pull Request
I butchered AI code to make it a feature for the manager camera. Testing shows that it essentially works but we should test it first due to the tracking datum being shared by sephirah and managers. 
DRV and meat lantern have a false on their track so they wont be able to be followed by the managers ability.
Two creatures with the same name will show up the same on the managers follow choices. It is randomized who you end up following. Dead agents will need to be either disfigured or cleaned up to prevent the system from getting confused.

Doubleclicking to follow code is hardcoded into AI mobs so unless we are allowed to make a signal for double click, which may be weird since we already have a signal for clicking, we may have to use more roundabout solutions.

## Why It's Good For The Game
Ive been requested to code this by a few manager players. Quality of life even if it is very rough in its development.

## Changelog
:cl:
add: follow ability for manager console
/:cl:

### CODE REVIEW

**Comment by Gboster-0** [comment]
there is a good chunk of single-letter variables and bare-bones comments on this, but otherwise looks good

File: code/game/machinery/computer/manager_camera.dm [hunk 1]
```diff
@@ -233,6 +244,88 @@
...
+
+	return targets
+
+/obj/machinery/computer/camera_advanced/manager/proc/ActualTrack(mob/living/target)
+	if(!istype(target))
+		to_chat(current_user, "<span class='warning'>Invalid Tracking Error.</span>")
+		return
+
+	tracking_subject = target
+	tracking = 1
```
Comment (Gboster-0) [inline code/game/machinery/computer/manager_camera.dm:282]: cant this be a `TRUE` instead of `1`?

**Comment by Kitsunemitsu** [comment]
I'm willing to merge this until a better solution is achieved!

**Comment by InsightfulParasite** [comment]
Simplified the code with a orbit proc that ghosts use. Downside is that the camera no longer loses targets automatically.

**Comment by InsightfulParasite** [comment]
I have two commits. The first one is the old code that used AI role code and the second is the new simplified one using orbit code. Im hesitant on merging both together since someone may say that the old code is better suited.

**Review by Kitsunemitsu — APPROVED** [review]
Saw video footage, works excellently

**Pull request merged**